demo/package.json

```diff
@@ -18,9 +18,10 @@
     "clean": "rimraf dist/ yarn-error.log",
     "quality": "run-s quality:*",
     "quality:test": "exit 0",
-    "quality:format": "prettier --write \"./**/*.{js,jsx,ts,tsx,css,scss,md,yaml,yml,json}\"",
+    "quality:format": "prettier --ignore-path ../.prettierignore --write \"./**/*.{js,jsx,ts,tsx,css,scss,md,yaml,yml,json}\"",
```

Comment:
Just out of curiosity... `ignore-path` is required but `--config` ain't?

Reply:
Each package has it's own prettier config that inherits from root config, so doesn't have to be specified explicitly.
But each package uses root prettierignore, because it can't be extended (afaik), and don't wanna copypaste

Reply:
I run some tests locally now. Prettier does a lookup for a the config file as it is not in every package file but it ain't doing it for the ignore file. So it is ok like it is now.